package.json

```diff
@@ -9,7 +9,7 @@
     "build-assets": "make js && make css && make components",
     "build-assets:css": "make css",
     "build-assets:js": "make js",
-    "svg-min": "svgo --config .svgo.yml static/images/**/*.svg static/images/*.svg",
+    "svg-min": "svgo --config conf/config/svgo.config.js static/images/**/*.svg static/images/*.svg",
```

Comment:
```suggestion
    "svg-min": "svgo --config conf/svgo.config.js static/images/**/*.svg static/images/*.svg",
```

Reply:
@cclauss I appreciate your patience - thank you for walking me through that.